processor/transformer/transformer.go

```diff
@@ -150,16 +146,15 @@ func GetVersion() string {
 	}
 	defer func() { httputil.CloseResponse(resp) }()
 
-	if resp.StatusCode == http.StatusOK {
-		bodyBytes, err := io.ReadAll(resp.Body)
-		if err != nil {
-			fmt.Println("Unable to read response into bytes with error : ", err.Error())
-			return "Unable to read response from transformer."
-		}
-
-		return string(bodyBytes)
+	if resp.StatusCode != http.StatusOK {
+		return defaultBuildVersion
+	}
+	bodyBytes, err := io.ReadAll(resp.Body)
+	if err != nil {
+		fmt.Println("Unable to read response into bytes with error : ", err.Error())
```

Comment:
Should we use a logger here?

Reply:
I wanted to get rid of this function too if we were okay with it

Reply:
I didn't see transformer_build_version set in either of user_transformer or dest_transformer pods enviornments in multi-tenant rudder-transformer and user-transformer. Not sure if its being set from elsewhere. 

Can potentially get rid of this altogether 👍